src/optimization_container_metadata.jl

```diff
@@ -0,0 +1,27 @@
+struct OptimizationContainerMetadata
+    container_key_lookup::Dict{String, <:OptimizationContainerKey}
+end
+
+function OptimizationContainerMetadata()
+    return OptimizationContainerMetadata(Dict{String, OptimizationContainerKey}())
+end
+
+function deserialize_metadata(
+    ::Type{OptimizationContainerMetadata},
+    output_dir::String,
+    model_name,
+)
+    filename = _make_metadata_filename(model_name, output_dir)
```

Comment:
_make_metadata_filename isn’t in this package.

Reply:
moved